src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categries',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      state.categories = state.categories.push('Under Construction');
```

Comment:
- [ ] You have done a great job on this milestone but there is a cogent issues i will like to point for you please kindly add action to the rducer checkstatus This is important because several reducers can update state from a single action. An action, is an object that contains the payload of information. They are the only source of information for the Redux store to be updated. Reducers update store based on the value of the action. see  example below

`reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
`